www/js/insert-slide.js

```diff
@@ -52,6 +57,17 @@ const langSlider = '<div class="lang-switch">'.concat(
   '</div>',
   '</td>',
   '</tr>',
+  '<tr>',
+  '<td>',
+  '<span class="lang-text">Show Announcement in Overlay</span>',
```

Comment:
the 2 switches could use some `padding-right` imo. 
![image](https://user-images.githubusercontent.com/2982784/64181784-fcaac680-ce1b-11e9-9bb8-fa17d36802e9.png)